samples/nodejs-compat/worker.js

```diff
@@ -0,0 +1,17 @@
+import { EventEmitter } from 'node:events';
```

Comment:
you can also easily add an import test to https://github.com/cloudflare/workerd/blob/main/src/workerd/api/node/node-test.c%2B%2B (maybe for async context?)

Reply:
Yeah, I'll be porting the tests we have in the internal ew-test files out to here separately